pkg/securitypolicy/framework.rego

```diff
@@ -264,6 +271,35 @@ signal_ok(signals) {
     input.signal == signal
 }
 
+plan9_mounted(target) {
+    data.metadata.p9mounts[target]
+}
+
+default plan9_mount := {"allowed": false}
+
+plan9_mount := {"p9mounts": p9mounts, "allowed": true} {
+    not plan9_mounted(input.target)
+    some containerID, _ in data.metadata.matches
+    pattern := concat("", [input.rootPrefix, "/", containerID, input.mountPathPrefix])
```

Comment:
Note the pattern matching here to ensure the target matches the template (as opposed to literal lists of targets as before)

Reply:
I know this currently linux focused, and plan9 is only linux, but are there plans to make filepath joining OS-neutral, so if we do want to port over to windows, we wont have to change a lot of the logic?